pkg/lib/feature/forgotpassword/errors.go

```diff
@@ -13,3 +13,5 @@ var SendCodeFailed = apierrors.Invalid.WithReason("ForgotPasswordFailed")
 
 var ErrFeatureDisabled = SendCodeFailed.NewWithCause("forgot password is disabled", apierrors.StringCause("FeatureDisabled"))
 var ErrUserNotFound = SendCodeFailed.NewWithCause("specified user not found", apierrors.StringCause("UserNotFound"))
+
+var ErrEmailIdentityNotFound = apierrors.Invalid.WithReason("EmailIdentityNotFound").NewWithCause("email identity not found", apierrors.StringCause("EmailIdentityNotFound"))
```

Comment:
According to the spec, it should https://github.com/authgear/authgear-server/pull/4510/files#diff-3be2ade1b2238264e843494f297f5b50d16774dcdee45eef65b8fe9de515ae4dR55.

And do not add cause for it. The reason is enough.